pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_population_data.py

```diff
@@ -344,35 +186,24 @@ def test_load_age_population_data(self, mock_read_excel1, mock_read_excel2, mock
 
         # TODO: How to test hdf5 export?
 
-    @patch('memilio.epidata.getPopulationData.gd.loadCsv')
-    @patch('memilio.epidata.getPopulationData.pandas.read_json')
-    @patch('memilio.epidata.getPopulationData.gd.loadExcel')
+    @ patch('memilio.epidata.getPopulationData.gd.loadCsv')
+    @ patch('memilio.epidata.getPopulationData.pd.read_json')
+    @ patch('memilio.epidata.getPopulationData.gd.loadExcel')
     def test_errors(self, mocklexcel, mockrjson, mocklcsv):
         mockrjson.side_effect = ValueError
-        [read_data, file_format, out_folder, no_raw] = [
-            True, "json", self.path, True]
-
-        directory = os.path.join(out_folder, 'Germany/')
-        gd.check_dir(directory)
-        with self.assertRaises(SystemExit) as cm:
-            gpd.get_one_data_set(read_data, file_format,
-                                 no_raw, directory, self.d1)
-        file_in = os.path.join(self.path, "Germany/FullDataB.json")
-        exit_string = "Error: The file: " + file_in + " does not exist. "\
-            "Call program without -r flag to get it."
-        self.assertEqual(cm.exception.code, exit_string)
-
         mocklexcel.side_effect = ValueError
+        mocklcsv.side_effect = ValueError
```

Comment:
Here are three error side effects introduced but only two errors are testet. At least one side effect seems to be not necessary.

Reply:
Removed the read_json mock.